newIDE/app/src/AssetStore/index.js

```diff
@@ -235,8 +235,9 @@ export const AssetStore = ({
                   tooltip={t`Back to discover`}
                   onClick={() => {
                     navigationState.openHome();
-                    setIsFiltersPanelOpen(false);
                     clearAllFilters(assetFiltersState);
+                    setIsFiltersPanelOpen(false);
+                    setIsAssetPackAdded(false);
                   }}
```

Comment:
After coming back to the pack, it will show as not already added.
Should this boolean be evaluated from what is in he object list?
Do we keep this for a next version?

Reply:
I was thinking of keeping for a next version.
Yes, the button will be enabled again, but the dialog opening up does the logic of comparing the assets and shows a warning "You already have this pack installed, do you want to install it again?"

Reply:
ok